src/Autocomplete/assets/test/browser/placeholder.test.ts

```diff
@@ -5,3 +5,48 @@ test('Can see homepage', async ({ page }) => {
 
     await expect(page.getByText("Symfony UX's E2E App")).toBeVisible();
 });
+
+test('disconnect/reconnect does not cause double-initialization error (issue #2623)', async ({ page }) => {
```

Comment:
I don't understand how this test could work, the file `apps/e2e/src/Controller/AutocompleteController.php` must be updated in order to render an autocomplete field

Reply:
This one is more tricky than initially thought. I was unable to come up with an example which would trigger this behaviour.
I will remove the test because it does not add any value. What will remain are the examples for the E2E-setup.